content/en/blogs/x509-certificate-exporter.md

```diff
@@ -0,0 +1,288 @@
+---
+title: 'Use x509-certificate-exporter to Monitor Component Certificates of a Kubernetes Cluster'  
+tag: 'Prometheus, KubeSphere'  
+keywords: x509-certificate-exporter, Prometheus, Kubernetes, Helm, KubeSphere, Certificate Monitoring 
+description: This article details how to deploy x509-certificate-exporter in Kubernetes and monitor component certificates of a Kubernetes cluster using a custom alerting policy on KubeSphere.
+createTime: '2021-06-30'  
+author: 'Yang Chuansheng, Bettygogo'  
+snapshot: '/images/blogs/en/x509-certificate-exporter/x509-certificate-exporter-cover-image.png'
+---
+
+KubeSphere offers an O&M friendly wizard that simplifies the O\&M of Kubernetes, but it is essentially built on Kubernetes. Kubernetes' TLS certificates are valid for only one year, so we need to update the certificates every year, which is unavoidable even though the cluster is installed by the powerful and lightweight installation tool [KubeKey](https://github.com/kubesphere/kubekey). To prevent possible risks arising from certificate expiration, we need to find a way to monitor certificate validity of Kubernetes components.
```

Comment:
Please use full names rather than abbreviations for `O&M` across the website. Or you can replace it with `developer-friendly`.

Reply:
Fixed, 3ks